webhooks/elbv2/targetgroupbinding_validator.go

```diff
@@ -182,6 +193,20 @@ func (v *targetGroupBindingValidator) getTargetGroupFromAWS(ctx context.Context,
 	return tgList[0], nil
 }
 
+func (v *targetGroupBindingValidator) getTargetGroupsByNameFromAWS(ctx context.Context, tgName string) (*elbv2sdk.TargetGroup, error) {
+	req := &elbv2sdk.DescribeTargetGroupsInput{
+		Names: awssdk.StringSlice([]string{tgName}),
+	}
+	tgList, err := v.elbv2Client.DescribeTargetGroupsAsList(ctx, req)
+	if err != nil {
+		return nil, err
+	}
+	if len(tgList) != 1 {
+		return nil, errors.Errorf("expecting a single targetGroup with name [%s] but got %v", tgName, len(tgList))
```

Comment:
The caller also includes the desired tgName, so consider removing that from this message.

Reply:
If one needs to read logs, one is usually not having a good day. More info means more helpful logs. I believe we should keep that. It's not sensitive at all.

Reply:
Duplicating the same information in a single log line is not more helpful.

Reply:
Sorry, why am I duplicating the same information in a single log line ? `tgName != len(tgList)`